packages/website/components/PageContent/PageContentHeader.tsx

```diff
@@ -57,15 +69,21 @@ export function PageContentHeader({
       </DisplayText>
 
       {github && (
```

Comment:
Shouldn't we remove this condition now?

This prop is not used by the things that are rendered if the condition is true

Reply:
Nice catch, forgot to remove it